lib/routes/crac/exam.ts

```diff
@@ -0,0 +1,49 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+
+export const route: Route = {
+    path: '/exam',
+    categories: ['government'],
+    example: '/exam',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '中国无线电协会业余无线电分会-考试信息',
+    maintainers: ['admxj'],
+    handler,
+};
+
+async function handler() {
+    const baseUrl = 'http://82.157.138.16:8091/CRAC';
+
+    const response = await got({
+        method: 'post',
+        url: `${baseUrl}/app/exam_advice/examAdviceList`,
+        body: { req: { type: '0', page_no: '1', page_size: '10000' } },
+    });
+
+    const list = response.data.res.list.map((item) => {
+        const id = Buffer.from(item.id).toString('base64');
+        const type = Buffer.from(item.type).toString('base64');
+        const link = `${baseUrl}/crac/pages/list_detail.html?id=${id}&type=${type}`;
+        return {
+            title: item.name,
+            link,
+            pubDate: item.createDate,
+            startDate: item.exam.signUpStartDate,
+            examType: item.examType,
```

Comment:
Useless properties. They are not listed in https://github.com/DIYgod/RSSHub/blob/459760046b18f2bcb886a99623cacd70fa93f3a6/lib/types.ts#L32